resources/js/toggle.js

```diff
@@ -0,0 +1,27 @@
+export class Toggle {
+    constructor(selector) {
+        this.selector = selector;
+        this.init();
+    }
+    init() {
+        document.querySelectorAll(this.selector).forEach(toggle => {
+            toggle.addEventListener('click', function() {
+                const content = toggle.parentNode.querySelector('.toggle-detail-content');
+                const arrows = toggle.querySelectorAll('.toggle-arrow');
+                if (content) {
+                    const isOpen = !content.classList.toggle('hidden');
+                    if (arrows.length === 2) {
+                        arrows[0].style.display = isOpen ? 'inline' : 'none'; // arrow-up
+                        arrows[1].style.display = isOpen ? 'none' : 'inline'; // arrow-down
+                    }
+                }
```

Comment:
@baovnt-zigexn
Nên sửa lại như bên dưới để hạn chế lồng if. 
```suggestion
                if (!content) return;
                
                const isOpen = !content.classList.toggle('hidden');
                if (arrows.length === 2) {
                    arrows[0].style.display = isOpen ? 'inline' : 'none'; // arrow-up
                    arrows[1].style.display = isOpen ? 'none' : 'inline'; // arrow-down
                }
```

Reply:
`isOpen` chỉ dùng trong scope của `if (arrows.length === 2)` nên gộp 2 điều kiện if thành 1 kiểu
```
if (content && arrows.length === 2) {
}
```